terminologies/technique/multiElectrodeRecording.jsonld

```diff
@@ -0,0 +1,8 @@
+{
+ "@id":"https://openminds.ebrains.eu/controlledTerms/technique/multiElectrodeRecording",

```

Comment:
do we want to have this consistent with the other terms? meaning "multi-electrode electrophysiology"?

Reply:
Marked in PanProject list